deployment/ansible/roles/common/tasks/unatended_upgrades.yml

```diff
@@ -0,0 +1,16 @@
+- name: Install unattended upgrades
```

Comment:
Should the file be removed if it's commented out in the main?

Reply:
This is here if we in the future want to migrate away from the LRZ.
I'd keep it, as this is likely (I am not entirely thrilled with the service we get and what “we” pay for it)

Reply:
I have added a clairifying comment